src/main/java/org/galatea/starter/service/IexService.java

```diff
@@ -60,6 +164,54 @@
    */
   public List<IexHistoricalPrice> getAllHistoricalPrices(final String symbol, final String range,
       final String date) {
-    return iexCloudClient.getAllHistoricalPrices(symbol, range, date);
+
```

Comment:
This is much better, but it can still be improved.

come over when you have a chance, its a bit hard to put into a pr comment